### PR TITLE
Feature/cms pages

### PR DESCRIPTION
**Pull request by tsubik**
Ability to add simple page with html content. We should migrate the rest of pages to use this simpler API.

### CODE REVIEW

File: app/admin/about_page_entries.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ def scoped_collection
       f.input :position, hint: "leaving empty will assign last position"
       f.input :code, hint: 'must be "partners" for Partners and "donors" for Donors section'
     end
-    f.translated_inputs switch_locale: false do |t|
+    f.translated_inputs "Translations", switch_locale: false do |t|
```
Comment (santostiago) [inline app/admin/about_page_entries.rb:54]: Shouldn't we localize this?

**Review by santostiago — APPROVED** [review]
All good!
I just left a comment regarding the localization of the word "Translations".

**Comment by santostiago** [comment]
Oh.. and there's some issue with the Linter.

File: app/admin/about_page_entries.rb (hunk 1 above)
Comment (tsubik) [reply app/admin/about_page_entries.rb:54]: The thing is that it's not showing up anywhere. This gem is unmaintained and the param must be there but is unused.

Comment (tsubik) [reply app/admin/about_page_entries.rb:54]: I removed it before, but realized that those options that are second parameter stopped working. Checked the gem code, and discovered that first parameter is not used.

**Pull request merged**